edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/IEdgeHub.cs

```diff
@@ -32,5 +32,7 @@ public interface IEdgeHub : IDisposable
         Task AddSubscription(string id, DeviceSubscription deviceSubscription);
 
         Task RemoveSubscription(string id, DeviceSubscription deviceSubscription);
+
+        Task ProcessSubscriptions(string id, IEnumerable<(DeviceSubscription, bool)> subscriptions);
```

Comment:
What's the difference between `AddSubscription` and `ProcessSubscriptions`?

Reply:
It is the same, except that ProcessSubscriptions takes a list (batch) of subscriptions, along with a flag to indicate whether to add or remove.
Maybe I can replace AddSubscription/RemoveSubscription with ProcessSubscription. That might make it clearer.